roles/etcd/templates/etcd.conf.j2

```diff
@@ -1,6 +1,18 @@
+{% if etcd_cluster -%}
+{% macro etcd_peers() -%}
+{% for host in groups[etcd_peers_group] -%}
+   {% if loop.last -%}
+"{{ hostvars[host]['ansible_fqdn']}}:{{ etcd_peer_port }}"
+   {%- else -%}
+"{{ hostvars[host]['ansible_fqdn']}}:{{ etcd_peer_port }}",
```

Comment:
I would probably just go ahead and create a new array of the hostnames and then use the join filter to insert the commas between the items.